src/containers/YAMLEditor/YAMLEditor.jsx

```diff
@@ -15,13 +15,14 @@ import {
   Button,
   Form,
   FormGroup,
-  InlineNotification,
-  Loading
+  InlineNotification
 } from 'carbon-components-react';
 import yaml from 'js-yaml';
 import React, { useEffect, useState } from 'react';
 import CodeMirror from '@uiw/react-codemirror';
-import { langs } from '@uiw/codemirror-extensions-langs';
+import { StreamLanguage } from '@codemirror/language';
+import { yaml as yamlMode } from '@codemirror/legacy-modes/mode/yaml';
```

Comment:
Switched back to the previous method of loading the YAML definition as the 'new' way results in problems with tree-shaking and unwanted additional language definition files being produced in the build output.

See comment below with fix (workaround) for previous problem with vitest.